Cloudflare/audit_logs/tests/test_gateway_create_location.json

```diff
@@ -0,0 +1,63 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Cloudflare Audit Logs",
+        "dialect_uuid": "76d767ed-5431-4db1-b893-a48b6903d871"
+      }
+    },
+    "message": "{\"ActionResult\":true,\"ActionType\":\"gateway_create_location\",\"ActorEmail\":\"sebastien.quioc@sekoia.fr\",\"ActorID\":\"4fbb09dc367b74f992786681022426c2\",\"ActorIP\":\"::ffff:78.197.123.35\",\"ActorType\":\"user\",\"ID\":\"5cd42473-600e-4610-a85e-ab194b8155ae\",\"Interface\":\"UI\",\"Metadata\":{\"Request\":{\"client_default\":false,\"linked_policies\":null,\"name\":\"AWS VM\",\"networks\":[{\"network\":\"15.188.186.81/32\"}]},\"account_id\":\"1d1e650b3385b95db72bba7cfb1287e9\",\"action_detail\":\"create gateway location\",\"actor_email\":\"sebastien.quioc@sekoia.fr\",\"location_name\":\"AWS VM\"},\"NewValue\":{\"name\":\"AWS VM\",\"networks\":[{\"network\":\"15.188.186.81/32\"}],\"linked_policies\":null,\"client_default\":false},\"OldValue\":{},\"OwnerID\":\"1d1e650b3385b95db72bba7cfb1287e9\",\"ResourceID\":\"a26c188f10c243338ed4c823fafa06f6\",\"ResourceType\":\"gateway.location\",\"When\":\"2023-02-24T15:09:53Z\"}"
+  },
+  "expected": {
+    "message": "{\"ActionResult\":true,\"ActionType\":\"gateway_create_location\",\"ActorEmail\":\"sebastien.quioc@sekoia.fr\",\"ActorID\":\"4fbb09dc367b74f992786681022426c2\",\"ActorIP\":\"::ffff:78.197.123.35\",\"ActorType\":\"user\",\"ID\":\"5cd42473-600e-4610-a85e-ab194b8155ae\",\"Interface\":\"UI\",\"Metadata\":{\"Request\":{\"client_default\":false,\"linked_policies\":null,\"name\":\"AWS VM\",\"networks\":[{\"network\":\"15.188.186.81/32\"}]},\"account_id\":\"1d1e650b3385b95db72bba7cfb1287e9\",\"action_detail\":\"create gateway location\",\"actor_email\":\"sebastien.quioc@sekoia.fr\",\"location_name\":\"AWS VM\"},\"NewValue\":{\"name\":\"AWS VM\",\"networks\":[{\"network\":\"15.188.186.81/32\"}],\"linked_policies\":null,\"client_default\":false},\"OldValue\":{},\"OwnerID\":\"1d1e650b3385b95db72bba7cfb1287e9\",\"ResourceID\":\"a26c188f10c243338ed4c823fafa06f6\",\"ResourceType\":\"gateway.location\",\"When\":\"2023-02-24T15:09:53Z\"}",
+    "@timestamp": "2023-02-24T15:09:53Z",
+    "user": {
+      "id": "1d1e650b3385b95db72bba7cfb1287e9"
+    },
+    "cloudflare": {
+      "ActionResult": true,
+      "ActionType": "gateway_create_location",
+      "ActorType": "user",
+      "ID": "5cd42473-600e-4610-a85e-ab194b8155ae",
+      "Interface": "UI",
+      "Metadata": {
+        "Request": {
+          "client_default": false,
+          "linked_policies": null,
+          "name": "AWS VM",
+          "networks": [
+            {
+              "network": "15.188.186.81/32"
+            }
+          ]
+        },
+        "account_id": "1d1e650b3385b95db72bba7cfb1287e9",
```

Comment:
I think this information may be duplicated in [`cloud.account.id`](https://www.elastic.co/guide/en/ecs/8.6/ecs-cloud.html#field-cloud-account-id)